src/libawkward/array/Record.cpp

```diff
@@ -52,7 +53,7 @@ namespace awkward {
     size_t cols = (size_t)numfields();
     std::shared_ptr<RecordArray::ReverseLookup> keys = array_.reverselookup();
     if (istuple()) {
-      keys = std::shared_ptr<RecordArray::ReverseLookup>(new RecordArray::ReverseLookup);
+      keys = std::make_shared<RecordArray::ReverseLookup>();
       for (size_t j = 0;  j < cols;  j++) {
```

Comment:
Isn't `make_shared` a C++14 thing?

Reply:
I was wrong! This is "[since C++11](https://en.cppreference.com/w/cpp/memory/shared_ptr/make_shared)". I guess I should have been using this, so that there are fewer dangerous-looking `new` keywords around.